notebooks/tuning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>I think in terms of ordering this section might make more sense before model backtesting. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>yes makes sense, will move it to end!</p>